skp_generator/lib/tests/page_flip.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/widgets.dart';
+
+class PageFlipTest extends StatelessWidget {
+  const PageFlipTest({
+    Key? key,
+    required this.angle,
+  }) : assert(angle >= 0.0),
+       assert(angle < 0.5),
+       super(key: key);
+
+  // 0.0, 0.097, 0.194
+  final double angle;
+
+  @override
+  Widget build(BuildContext context) {
+    return Transform(
+      transform: (Matrix4.rotationY(angle * math.pi)..setEntry(3, 0, angle * 0.008)) *
+                 Matrix4.diagonal3Values((1.0 - angle), (1.0 - angle), 1.0),
+      alignment: Alignment.center,
+      child: GridPaper(
+        interval: 256.0,
+        child: Container(
+          margin: EdgeInsets.all(20.0),
+          alignment: Alignment.center,
+          decoration: BoxDecoration(
+            color: const Color(0xFF006666),
+            border: Border.all(color: const Color(0xFF0000FF), width: 8.0),
+          ),
+          child: Text('My name\'s not Kirk... It\'s Skywalker.',
```

Comment:
nit: consider either `'''` or `"` instead of escaping the `'`.

Reply:
I find it easier to read code if all strings always use the same syntax and escaping than if there's different kinds of strings used everywhere. If it were up to me I'd ban use of `"`, `'''`, and probably even `r'` though maybe allowing it specifically for RegExps...